web/src/frontend/src/digisos/kvittering/index.tsx

```diff
@@ -41,38 +62,51 @@ class KvitteringView extends React.Component<
 		);
 	}
 	render() {
-		const { kvittering, restStatus, intl } = this.props;
+		const { kvittering, visVedlegg, restStatus, intl } = this.props;
 		return (
-			<LoadContainer restStatus={restStatus}>
-				<AppTittel />
-				<div className="kvittering skjema-content">
-					<div className="blokk-xl">
-						<Panel>
-							<Icon kind="stegindikator__hake" className="kvittering__ikon" />
-							<Undertittel className="kvittering__tittel">
-								{getIntlTextOrKey(intl, "kvittering.undertittel")}
-							</Undertittel>
-							{kvittering ? (
-								<div className="kvittering__tekst">
-									<p>
-										{getIntlTextOrKey(intl, "kvittering.tekst.pre")} {" "}
-										<strong>{kvittering.navenhet}</strong>
-										{getIntlTextOrKey(intl, "kvittering.tekst.post")}
-									</p>
-								</div>
-							) : null}
-						</Panel>
+			<LoadContainer
+				restStatus={restStatus}
+				contentRenderer={() => (
```

Comment:
Ah, skjønner problemet, kvitterings-delen blir jo fortsatt evaluert selv om spinneren spinner, så da tryner det om den er undefined. Akkurat der er angular mer elegant ass.

Reply:
Jupp. I og med det er en LoadContainer, burde den nesten kun tatt inn contentRenderer - per def er jo ikke innholdet klart for rendring før det er lastet